priv/templates/measurements.html.eex

```diff
@@ -66,10 +77,10 @@
   let schedulersByType = [
     <%= for {title, column} <- Enum.with_index(["Normal", "CPU", "IO"]) do %>
       <%
-        metric = as_downcase_atom("total_#{title}")
-        {new_seconds, new_percent} = new_measurements.scheduler_info[metric]
-        {base_seconds, base_percent} = if has_base, do: base_measurements.scheduler_info[metric], else: {0, 0}
-      %>
+     metric = as_downcase_atom("total_#{title}")
+     {new_seconds, new_percent} = new_measurements.scheduler_info[metric]
+     {base_seconds, base_percent} = if has_base, do: base_measurements.scheduler_info[metric], else: {0, 0}
+     %>
```

Comment:
Are you sure we should reformat this?

Reply:
I was searching for formatters to format .html.eex and haven’t found stable ones (Only custom defined or those built in Atom), and `mix format` can't handle those. The problem is, that it needs to format html parts first, and only then embedded elixir.  (A solution which I was trying to apply was: https://github.com/adamzapasnik/prettier-plugin-eex and still I'm not sure is it working correctly)

Reply:
Let's keep this version. We can discuss default formatter for `eex` files out of this PR.